.github/workflows/auto-pr-creation.yml

```
@@ -58,29 +58,46 @@ jobs:
+                      return issue_number, issue_title, None
+                  
+                  return None, None, None
               except subprocess.CalledProcessError as e:
                   print(f"Error finding issue: {e}")
                   return None, None, None
 
           def check_existing_pr(branch_name, base_branch):
               """Check if PR already exists for this branch"""
-              cmd = ['gh', 'pr', 'list', '--head', branch_name, '--base', base_branch, '--json', 'number']
+              cmd = ['gh', 'pr', 'list', '--head', branch_name, '--base', base_branch]
               
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)